### ISSUE TITLE
Can UPGDSED be used to load only one driver while rejecting all others?

### ISSUE THREAD

**Issue by lolcocks123** [hfiref0x/UPGDSED#34]
So I am using DIfferentSLIAuto and it works great!
It uses modified drivers (which are unsigned to run non SLI cards in SLI mode).

But with new games the problem is that their anti cheats do not allow games to be run when Windows is in test mode.
Even with full UPGDSED, anti cheats detect that testing mode is on.

So,
tl;dr, is it possible to load just one driver (nvlddmkm.sys) using UPGDSED while rejecting all others?

**Comment by Fyyre** [comment]
Due to the modification nvlddmkm.sys signing will no longer be valid.

The bigger problem would be: whatever anti-cheat detecting the presence of UPGDSED, most likely.  As UPGDSED does not run windows in 'test mode'.

What anti cheat is it ?


> So I am using DIfferentSLIAuto and it works great!
> It uses modified drivers (which are unsigned to run non SLI cards in SLI mode).
> 
> But with new games the problem is that their anti cheats do not allow games to be run when Windows is in test mode.
> Even with full UPGDSED, anti cheats detect that testing mode is on.
> 
> So,
> tl;dr, is it possible to load just one driver (nvlddmkm.sys) using UPGDSED while rejecting all others?

**Comment by lolcocks123** [comment]
Easy Anti-Cheat.
Even using UPGDSED, it detects that 'driver signature enforcement' is disabled.


> Due to the modification nvlddmkm.sys signing will no longer be valid.
> 
> The bigger problem would be: whatever anti-cheat detecting the presence of UPGDSED, most likely. As UPGDSED does not run windows in 'test mode'.
> 
> What anti cheat is it ?
> 
> > So I am using DIfferentSLIAuto and it works great!
> > It uses modified drivers (which are unsigned to run non SLI cards in SLI mode).
> > But with new games the problem is that their anti cheats do not allow games to be run when Windows is in test mode.
> > Even with full UPGDSED, anti cheats detect that testing mode is on.
> > So,
> > tl;dr, is it possible to load just one driver (nvlddmkm.sys) using UPGDSED while rejecting all others?

**Comment by lolcocks123** [comment]
Bump.

If loading a driver without "bcdedit -set nointegritychecks on" is impossible then please close this issue.
I will get my answer.
